src/routes/webhooks/topgg.ts

```diff
@@ -0,0 +1,37 @@
+import ApiRequest from '@lib/structures/api/ApiRequest';
+import ApiResponse from '@lib/structures/api/ApiResponse';
+import { Route, RouteOptions } from 'klasa-dashboard-hooks';
+import { TOKENS } from '@root/config';
+import { UserSettings } from '@lib/types/settings/UserSettings';
+import { ApplyOptions } from '@skyra/decorators';
+
+@ApplyOptions<RouteOptions>({ name: 'webhooks/topgg', route: 'webhooks/topgg' })
+export default class extends Route {
+
+	public async post(request: ApiRequest, response: ApiResponse) {
+		if (request.headers.authorization !== TOKENS.WEBHOOK_B4D) return response.forbidden();
```

Comment:
```suggestion
		if (request.headers.authorization !== TOKENS.WEBHOOK_TOPGG) return response.forbidden();
```

Have you learned nothing from Lastpass stuff (╯°□°）╯︵ ┻━┻ unique passwords per server mate.
(config on vps has been updated)